src/Microsoft.OData.Core/OpenTelemetryContants.cs

```diff
@@ -0,0 +1,25 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="OpenTelemetryConstants.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+namespace Microsoft.OData
+{
+    internal class OpenTelemetryConstants

```

Comment:
I think we can remove references to "OpenTelemetry" in the core library since we're using `System.Diagnostics*`

```suggestion
    internal class TelemetryConstants
```